spring-cloud-gcp-autoconfigure/src/main/java/com/google/cloud/spring/autoconfigure/firestore/GcpFirestoreEmulatorAutoConfiguration.java

```diff
@@ -79,7 +79,8 @@ public FirestoreOptions firestoreOptions() {
   private Credentials emulatorCredentials() {
     final Map<String, List<String>> headerMap = new HashMap<>();
     headerMap.put("Authorization", Collections.singletonList("Bearer owner"));
-    headerMap.put("google-cloud-resource-prefix", Collections.singletonList(rootPath));
+    headerMap.put("google-cloud-resource-prefix", Collections.singletonList(
+        rootPath.substring(0, rootPath.indexOf("/documents"))));
```

Comment:
Is it possible for the database ID to be `documents`?

What about `lastIndexOf` instead?